### ISSUE TITLE
cauldron recipe don't work with JEI 10

### ISSUE THREAD

**Issue by ComeBelin** [JoeFoxe/Hexerei-0.2.1-1.18#65]
hello, I play on version 1.18.2 and as JEI 10 has just been released Hexerei does not work correctly with it because the craft with the cauldron does not display correctly (the frame remains empty without the texture of the mod's craft)

**Comment by LykiaOokami** [comment]
Yep, got the same issue - 1.18.2. and JEI 10 shows no recipe at all, prior JEI versions crash with Forge 1.18.2. so I'm stuck with no recipes
